pykern/pkinspect.py

```diff
@@ -116,7 +116,7 @@ def caller(ignore_modules=None, exclude_first=True):
             del frame
 
 
-def caller_module():
+def caller_module(exclude_first=True):
```

Comment:
Description needs to be changed to describe exclude_first.

Reply:
Needs a test

Reply:
b29cdf7